shifterator/plotting.py

```diff
@@ -1,132 +1,83 @@
-# -*- coding: utf-8 -*-
 """
-plotting.py
-
 TODO:
-- Add params for explicitly setting fonts
+ - Add params for explicitly setting fonts
 """
 import numpy as np
 from matplotlib import rcParams
 
+
 def get_plot_params(plot_params, show_score_diffs):
-    """
-    Sets default plotting values for shift graphs that are not set by the user
+    defaults = {
+        "all_pos_contributions": False,
+        "alpha_fade": 0.35,
+        "bar_linewidth": 0.25,
+        "bar_type_space_scaling": 0.015,
+        "bar_width": 0.8,
+        "cumulative_xlabel": None,
+        "cumulative_xticklabels": None,
+        "cumulative_xticks": None,
+        "cumulative_ylabel": None,
+        "detailed": True,
+        "dpi": 200,
+        "height": 15,
+        "invisible_spines": [],
+        "label_fontsize": 13,
+        "missing_symbol": "*",
+        "pos_cumulative_inset": [0.19, 0.12, 0.175, 0.175],
+        "pos_text_size_inset": [0.81, 0.12, 0.08, 0.08],
+        "remove_xticks": False,
+        "remove_yticks": False,
+        "score_colors": {
+            "all_pos_neg": "#C4CAFC",
+            "all_pos_pos": "#FFFF80",
+            "neg_s": "#9E75B7",
+            "neg_s_neg_p": "#C4CAFC",
+            "neg_s_pos_p": "#2F7CCE",
+            "neg_total": "#C4CAFC",
+            "pos_s": "#FECC5D",
+            "pos_s_neg_p": "#FDFFD2",
+            "pos_s_pos_p": "#FFFF80",
+            "pos_total": "#FFFF80",
+            "total": "#707070",
+        },
+        "serif": False,
+        "show_score_diffs": show_score_diffs,
+        "show_total": True,
+        "symbols": {
```

Comment:
Note here that in resolving merge conflicts, a little bit got lost in translation. 

I think it should be:
```
'pos_total': plot_params['system_names'][1],	
'neg_total': plot_params['system_names'][0],	
'all_pos_pos': plot_params['system_names'][1],	
'all_pos_neg': plot_params['system_names'][0],
```

Does this happen to resolve #19 if you make this change?

Reply:
Actually I'm looking locally and I was doing something a bit more complicated. 

```python
if diff >= 0:
    pos_sys_name = plot_params['system_names'][1]
    neg_sys_name = plot_params['system_names'][0]
else:
    pos_sys_name = plot_params['system_names'][0]
    neg_sys_name = plot_params['system_names'][1]
    plot_params['symbols'] = {
        'pos_s_pos_p': u'+\u2191',
        'pos_s_neg_p': u'+\u2193',
        'neg_s_pos_p': u'-\u2191',
        'neg_s_neg_p': u'-\u2193',
        'pos_s': u'\u25B3',
        'neg_s': u'\u25BD',
        'pos_total': pos_sys_name,
        'neg_total': neg_sys_name,
        'all_pos_pos': plot_params['system_names'][1],
        'all_pos_neg': plot_params['system_names'][0],
        'total': r'$\Sigma$'}
```

`diff` had to be added to the `get_plot_params` signature, and it was passed `self.diff` when it's called.

I think I hadn't decided if this was the right fix, which is why I hadn't pushed it yet. Let me know your thoughts on this.

Reply:
#19 is gone with these changes!

Reply:
As for the second snippet, I think it's the same as what I have for all of my test cases, so if it's more correct or nicer to work with for some of the cases that you have then I don't think there's much of a reason to not have it. I'll roll it into the PR.